app.json

```diff
@@ -3,31 +3,30 @@
     "name": "droidconKE2022ReactNative",
     "slug": "droidconKE2022ReactNative",
     "version": "1.0.0",
+    "owner": "reactdevske-reactnative",
     "orientation": "portrait",
-    "icon": "./assets/icon.png",
+    "icon": "./src/assets/icon.png",
```

Comment:
The paths to these assets were wrong, I wonder if they were working for you.

Reply:
Oops! Hadn't noticed this!